torch/package/package_exporter.py

```diff
@@ -690,7 +687,7 @@ def register_extern_hook(self, hook: ActionHook) -> RemovableHandle:
         Hooks will be called in order of registration.
 
         Returns:
-            :class:`torch.utils.hooks.RemovableHandle`:
+            :class:`torch.package._hooks.RemovableHandle`:
```

Comment:
This class should be a part of the public API, and thus should not have any underscores in its qualified name. The easiest way to do this is to re-export the name in `torch/pacakge/__init__.py` and reference it as `torch.package.RemovableHandle`.